src/composables/use-media-query.ts

```diff
@@ -16,7 +20,7 @@ interface Options {
  * Reactive Media Query.
  */
 export function useMediaQuery(
-  query: string,
+  query: MaybeComputedRef<string>,
```

Comment:
To update the desktop threshold based on the `new_header` flag status, we need to pass a computed value here.